sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2341,6 +2341,118 @@ class Dataset[T] private[sql](
     }
   }
 
+  /**
```

Comment:
the line wrapping here looks like you've used a shorter line length than the other code in this class

Reply:
fixed